draft-ietf-tls-tls13.md

```diff
@@ -3593,6 +3548,127 @@ appropriate application traffic key as described in {{updating-traffic-keys}}.
 In particular, this includes any alerts sent by the
 server in response to client Certificate and CertificateVerify messages.
 
+## 0-RTT and Anti-Replay {#replay-time}
```

Comment:
Not that it matters much, but should this section be labeled `{#anti-replay}` or something else instead? The old `{#replay-time}` section was named that when it only described the stateless time-mismatch mechanism.